### PR TITLE
add: careerup website and contract

### PR DESCRIPTION
**Pull request by joeephwild**
made some changes to the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +66,7 @@
 - [The Public Lock Contract](https://thirdweb.com/unlock-protocol.eth/PublicLock) - The Public Lock Contract is a membership contract from Unlock Protocol. This contracts lets you create recurring subscriptions, tickets, certifications.
 - [Rental NFTs](https://thirdweb.com/doubledev.eth/ERC4907) - ERC-4907 facilitates the efficient implementation of rentable NFTs.
 - [ERC-721 Community Stream](https://thirdweb.com/flairsdk.eth/ERC721CommunityStream) - Allows you to create a token stream to equally distribute any tokens (Native, ERC20, etc) to holders of a specific ERC721 NFT collection.
+- [Job Application](https://thirdweb.com/goerli/0xAa3C537CDeeFc8Ea9e86b7C5dAecF11fa4cDA01E) - job application smart contract allow users to list ajob, apply for a job.
```
Comment (warengonzaga) [inline README.md]: Hey @joeephwild maybe use a release page version of your contract to be able to use by the community.

**Review by warengonzaga — CHANGES_REQUESTED** [review]
Just replace the link with the release page version of your contract and I'm happy to merge it.

File: README.md (hunk 1 above)
Comment (joeephwild) [reply README.md]: okay sure will do that and rerun the pull request

Comment (joeephwild) [reply README.md]: im trying to update the code for better usage

Comment (warengonzaga) [reply README.md]: will wait for your PR @joeephwild

Comment (joeephwild) [reply README.md]: done

**Review by warengonzaga — APPROVED** [review]
LGTM!

**Pull request merged**